src/js/components/homepageUpdate/heroUpdate/HeroUpdate.jsx

```diff
@@ -44,7 +44,7 @@ const HeroUpdate = () => {
         }, 50);
         window.addEventListener('resize', handleResize);
         return () => window.removeEventListener('resize', handleResize);
-    }, []);
+    }, [windowWidth]);
```

Comment:
did you add this intentionally?

Reply:
I did not. But I decided not to remove it bc it seems like it should be there, it's a real dependency. We've left it in a couple of other components for this same useEffect and it isn't breaking anything.

Reply:
okay i just figured i'd check